client/app/scripts/superdesk-legal-archive/legal_archive.js

```diff
@@ -115,12 +119,23 @@
         };
     }
 
-    LegalArchiveController.$inject = ['$scope', '$location', 'legal'];
-    function LegalArchiveController($scope, $location, legal) {
+    LegalArchiveController.$inject = ['$scope', '$location', 'legal', 'preferencesService'];
+    function LegalArchiveController($scope, $location, legal, preferencesService) {
+        var viewUpdate = {'archive:view': {
+                            'allowed': ['mgrid', 'compact'],
+                            'category': 'archive',
+                            'view': 'mgrid',
```

Comment:
Not sure. Since Monitoring uses list view I will change it to List View as default view.